summary_page.php

```diff
@@ -153,7 +153,11 @@
 		<table class="table table-hover table-bordered table-condensed table-striped">
 		<thead>
 			<tr>
+			<?php if( 1 == count( $t_project_ids ) ) { ?>
+				<th class="width-35"><?php echo lang_get( 'email_project' ) ?></th>
```

Comment:
It's confusing to use `email_project` for something that is not related to email.
I would introduce a new string for it.